libff/CMakeLists.txt

```diff
@@ -55,11 +55,14 @@ target_link_libraries(
   ff
 
   ${GMP_LIBRARIES}
-  ${GMPXX_LIBRARIES}
-  ${CRYPTO_LIBRARIES}
   ${PROCPS_LIBRARIES}
   ${FF_EXTRALIBS}
 )
+target_include_directories(
```

Comment:
What is the motivation for including over linking GMP? There's currently a linking error in this build: https://travis-ci.org/scipr-lab/libff/builds/269181457#L1262

Reply:
You need both (unless you use CMake's imported targets).
I think the issue is the variable `GMP_LIBRARIES` is never set. I will check it.

Reply:
I think the builds are failing because the GMP was previously pulled by additional libcrypto library.